packages/paste-design-tokens/tokens/global/text-color.yml

```diff
@@ -437,3 +437,33 @@ props:
     comment: Twilio brand icon color used for the Twilio logo on inverse backgrounds.
     text_contrast_pairing:
       - color-background-body-inverse
+
+  # decorative
+  color-text-decorative-10:
```

Comment:
question: These text color tokens are automatically exposed into Box and Text, meaning anyone can use those text colors, anywhere in Twilio products. Are we OK with that? I guess that is a @twilio-labs/design-systems-pd question

Reply:
Yep, I'm good with that!